CMakeLists.txt

```diff
@@ -1,22 +1,28 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
+
 project(nmea_msgs)
 
-find_package(catkin REQUIRED COMPONENTS std_msgs message_generation)
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
 
-add_message_files(
-  FILES
-  Gpgga.msg
-  Gpgsa.msg
-  Gpgsv.msg
-  GpgsvSatellite.msg
-  Gprmc.msg
-  Sentence.msg
-)
+find_package(tinyxml2 REQUIRED)
```

Comment:
I can't get this to build in a minimal docker container. Here's what I did:

```bash
docker pull osrf/ros:crystal-desktop
docker run -it osrf/ros:crystal-desktop
apt-update && apt-install python3-colcon-common-extensions
mkdir -p code/ros2_ws/src
cd ros2_ws/src
git clone --branch ros2 https://github.com/klintan/nmea_msgs.git
cd ..
rosdep install --from-paths src
colcon build
```

The build fails with a missing dependency: tinyxml2

When I remove this `find_package`, the build succeeds. I don't think you need this dependency for anything, do you?

Reply:
Yeah it wasn't working on my local mac without it, but I also thought it was really strange. So I will remove it.